projects/fsastorefrontlib/src/cms-components/questionnaire-carousel/questionnaire-carousel.component.html

```diff
@@ -84,3 +87,15 @@ <h4 class="mb-4">{{ product.name }}</h4>
 <ng-template #finalResult>
   <h3>{{ 'fscommon.finalResult' | cxTranslate }}</h3>
 </ng-template>
+
+<ng-template #noResults>
+  <h5 class="notice-text text-center p-4">
+    <p class="m-0">
+      {{ 'fscommon.questionnaire.noResults' | cxTranslate }}
+
+      <a class="overview-section-link" routerLink="/agent-locator">{{
```

Comment:
Just `link` instead of `overview-section-link`.

Reply:
fixed